client/flutter/lib/generated/intl/messages_all.dart

```diff
@@ -15,45 +15,25 @@ import 'package:intl/intl.dart';
 import 'package:intl/message_lookup_by_library.dart';
 import 'package:intl/src/intl_helpers.dart';
 
-import 'messages_ru_RU.dart' as messages_ru_ru;
```

Comment:
Why are you removing theses ?

Reply:
this is out of scope for this release (https://github.com/WorldHealthOrganization/app/issues/765) and the files were just (outdated) MT'ed versions

Reply:
we're locking the app down to only english and french for this week's release

Reply:
I'll put back files with content machine translated after the release